packages/components/src/components/form/SelectBar/SelectBar.tsx

```diff
@@ -63,14 +68,16 @@ const Options = styled.div<{
     $optionsCount: number;
     $isFullWidth?: boolean;
     $elevation: Elevation;
+    $fillType: FillType;
 }>`
     position: relative;
     display: grid;
     grid-auto-columns: ${({ $optionsCount }) => `minmax(${getPuckWidth($optionsCount)}, 1fr)`};
     grid-auto-flow: column;
     gap: ${spacingsPx.xxs};
     padding: ${spacingsPx.xxs};
-    background: ${mapElevationToBackground};
+    background: ${({ $fillType, theme, $elevation }) =>
```

Comment:
Do it the other way around, unset the background when fillType is none. Then you don't have to pass all the props to the mapElevationToBackground.